@xen-orchestra/lite/src/components/AppLogin.vue

```diff
@@ -19,21 +28,51 @@
 
 <script lang="ts" setup>
 import { storeToRefs } from "pinia";
-import { onMounted, ref } from "vue";
+import { onMounted, ref, watch } from "vue";
+import { useI18n } from "vue-i18n";
+import FormInput from "@/components/form/FormInput.vue";
+import FormInputWrapper from "@/components/form/FormInputWrapper.vue";
 import UiButton from "@/components/ui/UiButton.vue";
 import { useXenApiStore } from "@/stores/xen-api.store";
 
+const { t } = useI18n();
 const xenApiStore = useXenApiStore();
 const { isConnecting } = storeToRefs(xenApiStore);
 const login = ref("root");
 const password = ref("");
+const error = ref<string>();
+const passwordRef = ref();
+const isInvalidPassword = ref(false);
+
+const focusPasswordInput = () =>
+  passwordRef.value?.$el.querySelector("input").focus();
 
 onMounted(() => {
   xenApiStore.reconnect();
+  focusPasswordInput();
 });
 
+watch(
+  () => password.value,
```

Comment:
ℹ️ Just for information (no need to fix), when watching a `Ref` or a `ComputedRef`, you can also pass it directly to `watch`:

```suggestion
  password,
```